bootstrap/velocity/src/main/java/org/geysermc/geyser/platform/velocity/command/VelocityCommandSource.java

```diff
@@ -84,4 +94,9 @@ public String locale() {
     public boolean hasPermission(String permission) {
         return handle.hasPermission(permission);
     }
+
+    @Override
+    public Object handle() {
```

Comment:
Wonder if we can make this a <T> sort of situation.

Reply:
generics won't work as far as i know, but i added some fallbacks in case the handle isn't assignable

Reply:
oh lol i get what you mean now. yes, could have done that and casted within the `handle()` method. the new code doesn't require unsafe casting anymore though